src/foam/nanos/u2/navigation/ResponsiveTopNav.js

```diff
@@ -84,6 +87,7 @@ foam.CLASS({
         .show(this.loginSuccess$)
         .addClass(this.myClass())
         .start().addClass(this.myClass('components-container'))
+          .addClass(this.myClass('logo-adjust'))
```

Comment:
Why do we need this?

Reply:
Whitespace between the hamburger button and logo doesn't match without it. I didn't want to do it for both `components-container`s because the buttons on the right seemed to look fine